<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPackageInputClass</class>
 <widget class="QMainWindow" name="CPackageInputClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPackageInput</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actionMatchStr"/>
    <addaction name="actionWriteInFile"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuickMatch">
   <property name="text">
    <string>QuickMatch</string>
   </property>
  </action>
  <action name="actionMatchStr">
   <property name="text">
    <string>MtachStr</string>
   </property>
  </action>
  <action name="actionWriteInFile">
   <property name="text">
    <string>WriteInFile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cpackageinput.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>ShowDialogMatch()</slot>
 </slots>
</ui>
